airbyte-integrations/connectors/source-salesforce/source_salesforce/rate_limiting.py

```diff
@@ -126,6 +128,11 @@ def interpret_response(self, response: Optional[Union[requests.Response, Excepti
             f"An error occurred: {response.content.decode()}",
         )
 
+    @staticmethod
+    def _is_bulk_job_status_check(response: requests.Response) -> bool:
+        # TODO comment on PR: I don't like that because it duplicates the format of the URL but with a test at least we should be fine to valide once it changes
+        return bool(re.compile(r"/services/data/v\d{2}\.\d/jobs/query/[^/]+$").search(response.url))
```

Comment:
Abort and delete both have the same URL but their methods are different. Should we match on the HTTP method POST as well?